src/scripts/condor_submit.sh

```diff
@@ -201,19 +201,27 @@ submit_file_environment="#"
 
 if [ "x$environment" != "x" ] ; then
 # Input format is suitable for bourne shell style assignment. Convert to
-# old condor format (no double quotes in submit file).
-# FIXME: probably it's better to convert everything into the 'new' Condor
-# environment format.
+# new condor format to avoid errors  when things like LS_COLORS (which 
+# has semicolons in it) get captured
     eval "env_array=($environment)"
     submit_file_environment=""
     for  env_var in "${env_array[@]}"; do
         if [ "x$submit_file_environment" == "x" ] ; then
-            submit_file_environment="environment = "
+            submit_file_environment="environment = \" ${env_var}"
```

Comment:
Probably minor, but is the space here between `\"` and `${env_var}"` intentional?  I'd guess no but it seems worth asking.